.github/workflows/main.yml

```diff
@@ -13,17 +13,17 @@ jobs:
 
     strategy:
       matrix:
-        python-version: ["3.6", "3.7", "3.8", "3.9"]
+        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10.0-alpha.7"]
```

Comment:
Is there a better way to do this? I want it to detect 3.10.0-beta* whenever they appear.

Reply:
Resolved by using [deadsnakes/action ](https://github.com/deadsnakes/action)